freqtrade/rpc/api_server.py

```diff
@@ -97,17 +95,14 @@ def stop():
         Section to handle configuration and running of the Rest serve
         also to check and warn if not bound to 127.0.0.1 as a security risk.
         """
+        rest_ip = self._config['api_server']['listen_ip_address']
+        rest_port = self._config['api_server']['listen_port']
 
-        rest_ip = self._config['rest_cmd_line']['listen_ip_address']
-        rest_port = self._config['rest_cmd_line']['listen_port']
-
-        if rest_ip != "127.0.0.1":
-            i=0
-            while i < 10:
-                logger.info("SECURITY WARNING - Local Rest Server listening to external connections")
-                logger.info("SECURITY WARNING - This is insecure please set to 127.0.0.1 in config.json")
-                i += 1
+        if not IPv4Address(rest_ip).is_loopback :
+            logger.info("SECURITY WARNING - Local Rest Server listening to external connections")
+            logger.info("SECURITY WARNING - This is insecure please set to your loopback, e.g 127.0.0.1 in config.json")
 
         # Run the Server
         logger.info('Starting Local Rest Server')
         app.run(host=rest_ip, port=rest_port)
```

Comment:
i guess you'll want to have some exception handling here should the port already be occupied?

Reply:
added